sdk/core/Azure.Core/src/Buffers/StreamMemoryExtensions.cs

```diff
@@ -14,9 +14,11 @@ internal static class AzureBaseBuffersExtensions
     {
         public static async Task WriteAsync(this Stream stream, ReadOnlyMemory<byte> buffer, CancellationToken cancellation = default)
         {
-            if (stream == null) throw new ArgumentNullException(nameof(stream));
+            if (stream == null)
+                throw new ArgumentNullException(nameof(stream));
 
-            if (buffer.Length == 0) return;
```

Comment:
what rule controls this. I feel this should be fine.

Reply:
That should be 
csharp_preserve_single_line_statements = false # vs-default: true
Should I, I'll go ahead and set that tot true.

Reply:
Isn't it csharp_preserve_single_line_blocks ?

Reply:
Well they are similar, but this particular case is  cause by `csharp_preserve_single_line_statements = false`. Actually `csharp_preserve_single_line_blocks = true` is set to true.

Reply:
You are right.

Reply:
Let's keep it as is and just fix the worst offenders like:
``` C#
if (read == 0)
{ break; }
```

Reply:
We should re-sync with corefx an pull in new things like `csharp_prefer_braces`.

Reply:
Do you want me to add all the rules that don't currently exist in our own .editorconfig?

Reply:
Not as part of this PR, but later - yes.